vmdb/app/controllers/ems_common.rb

```diff
@@ -219,11 +219,11 @@ def form_field_changed
           # Hide/show C&U credentials tab
           page << "$('metrics_li').#{params[:server_emstype] == "rhevm" ? "show" : "hide"}();"
         end
-        if ["openstack"].include?(params[:server_emstype])
+        if ["openstack", "openstack_infra"].include?(params[:server_emstype])
```

Comment:
@martinpovolny, I had the same thought the first time I touched that code.  And, still I haven't done anything about it :crying_cat_face:.

I agree with @tzumainn and @Fryguy ... this should definitely be fixed, but probably in a larger refactoring effort, likely tied to pluggable providers.